hack/verify-dirty.sh

```diff
@@ -9,19 +9,20 @@ echo "Verify uncommitted files..."
 echo "#############################"
 
 # Option to ignore changes to compatibility image path in
-# pkg/v1/tkg/tkgconfigpaths/zz_bundled_default_bom_files_configdata.go. This is to support `usebom` directive
+# tkg/tkgconfigpaths/zz_bundled_default_bom_files_configdata.go and plugin runtime version in
+# pkg/v1/cli/version/zz_generated_plugin_runtime_version.go. This is to support `usebom` directive
 # in tests.
-ignore_file=':!pkg/v1/tkg/tkgconfigpaths/zz_bundled_default_bom_files_configdata.go'
+ignore_files=(':(exclude)tkg/tkgconfigpaths/zz_bundled_default_bom_files_configdata.go' ':(exclude)pkg/v1/cli/version/zz_generated_plugin_runtime_version.go')
```

Comment:
On the `release-0.25` branch, the tkg file is still under `pkg/v1` so I think the new line should be:

`ignore_files=(':(exclude)pkg/v1/tkg/tkgconfigpaths/zz_bundled_default_bom_files_configdata.go' ':(exclude)pkg/v1/cli/version/zz_generated_plugin_runtime_version.go')`

Reply:
Thanks for catching this!